jira/jira-issues-to-dust.ts

```diff
@@ -37,7 +37,7 @@ if (missingEnvVars.length > 0) {
   );
 }
 
-const DUST_RATE_LIMIT = 120; // requests per minute
+const DUST_RATE_LIMIT = 118; // requests per minute
```

Comment:
When importing 3k JIRA tickets with the script I got ~20 `Error upserting issue SP-8658 to Dust datasource: AxiosError: Request failed with status code 429` : reducing the speed by 2% should give the margin needed to avoid these.